python/ray/_private/runtime_env/pip.py

```diff
@@ -493,6 +497,11 @@ def modify_context(
                 "installing the runtime_env `pip` packages."
             )
         context.py_executable = virtualenv_python
-        context.command_prefix += [
-            _PathHelper.get_virtualenv_activate_command(target_dir)
-        ]
+        if _WIN32:
+            context.command_prefix += _PathHelper.get_virtualenv_activate_command(
+                target_dir
+            )
+        else:
+            context.command_prefix += [
+                _PathHelper.get_virtualenv_activate_command(target_dir)
+            ]
```

Comment:
Should we make `get_virtualenv_activate_command` always return `List[str]` to avoid this if-else?

Reply:
Yeah I was thinking more about it. A cleaner solution would to have always add lists to the `command_prefix` item and then do the combination of the items to string for Linux/Mac and keep it as list items for Windows. So we would have to update it for each of the three plugins.